lib/Mojo/Message.pm

```diff
@@ -97,10 +97,14 @@ sub fix_headers {
   # Content-Length or Connection (unless chunked transfer encoding is used)
   my $content = $self->content;
   my $headers = $content->headers;
+  my $websocket = $headers->sec_websocket_accept;
+  my $isWebsocket = defined $websocket;
```

Comment:
perl style of variables naming is `$is_websocket`, not `$isWebsocket`

Reply:
done. thx